ocamldoc/odoc_analyse.ml

```diff
@@ -89,7 +89,7 @@ let process_implementation_file sourcefile =
       begin match Location.error_of_exn exn with
       | Some (`Ok err) ->
           fprintf Format.err_formatter "@[%a@]@."
-            Location.print_report err
+            (fun ppf -> Location.print_report (Misc.Log.Direct ppf)) err
```

Comment:
This is the only place I can see in the diff (so probably in the compiler) that calls `fprintf` around `print_report` in this way, which looks like a (pre-existing) mild code smell. Could we just call `print_report` here without the boxes around?

Reply:
Calling `print_report` directly is probably better here indeed.